packages/cli/generators/controller/index.js

```diff
@@ -207,12 +207,11 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
     if (this.shouldExit()) return false;
     this.artifactInfo.className = utils.toClassName(this.artifactInfo.name);
     this.artifactInfo.outFile =
-      utils.kebabCase(this.artifactInfo.name) + '.controller.ts';
+      utils.toFileName(this.artifactInfo.name) + '.controller.ts';
     if (debug.enabled) {
       debug(`Artifact output filename set to: ${this.artifactInfo.outFile}`);
     }
-
-    this.artifactInfo.modelVariableName = utils.camelCase(
+    this.artifactInfo.modelVariableName = utils.utilCamelCase(
```

Comment:
Why do we rename it to `utilCamelCase`?

Reply:
I think it should be named `toVarName` to be consistent with:
- toClassName
- toFileName

Reply:
true :+1::skin-tone-3: